### PR TITLE
feat(label): create `PLabel` component

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- <del>[ ] Tested with console</del> 
- Currently, `npm link` does not work between console and mirinae
- **Please check on console with released mirinae**

### Description
- Create `PLabel` component.
- Create `PLabel` Docs.

screencapture-localhost-6006-2022-11-17-15_37_56

### CODE REVIEW

File: src/data-display/label/PLabel.vue [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    .delete-icon {
+        @apply text-gray-400;
+        cursor: pointer;
+        margin-left: 0.25rem;
+    }
+
+    &.clickable {
+        cursor: pointer;
+
+        &:hover {
```
Comment (yuda110) [inline src/data-display/label/PLabel.vue:123]: for mobile UI :) (https://www.kenrhee.com/til/media-hover)

```suggestion
        @media (hover: hover) {
            &:hover {
```

File: src/data-display/label/PLabel.vue [hunk 2]
```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="p-label"
+         :class="{ clickable, active: proxyActive, iconLabel: leftIcon }"
```
Comment (yuda110) [inline src/data-display/label/PLabel.vue:3]: It would be nice to change class name `iconLabel` to `icon-label`

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/data-display/label/PLabel.vue (hunk 1 above)
Comment (piggggggggy) [reply src/data-display/label/PLabel.vue:123]: Oh.. I'm learning...

**Pull request merged**